### PR TITLE
Issue179 consistence with adc

### LINKED ISSUE #179: Ensure consistence between adc.met.no and the data management handbook
Currently, the translations in py-mmd-tools and thus the data management handbook are a bit out of sync with the ADC ones (see https://adc.met.no/node/4). We need to check and make sure that the two sources of information are consistent.

### PR DESCRIPTION
**Pull request by mortenwh**
**Summary**: closes #179 

**Related issue**: #179 

**Suggested reviewer(s)**:

**Reviewer checklist**:

- [ ] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
- [ ] 100% test coverage of new code - meaning:
    - [ ] The overall test coverage increased or remained the same as before
    - [ ] Every function is accompanied with a test suite
    - [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
    - [ ] Functions with optional arguments have separate tests for all options
- [ ] Examples are supported by doctests
- [ ] All tests are passing
- [ ] All names (e.g., files, classes, functions, variables) are explicit
- [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: py_mmd_tools/mmd_elements.yaml [hunk 1]
```diff
@@ -154,12 +198,14 @@ geographic_extent:
...
           minOccurs: '1'
           comment: No repetition allowed.
+          description: Describes a simple longitude limit; may be part of a 2- or 3-dimensional bounding region. geospatial_lon_min specifies the westernmost longitude covered by the dataset. See also geospatial_lon_max. Must be decimal degrees east (negative westwards).
   polygon:
     maxOccurs: '1'
     minOccurs: '0'
     acdd:
       geospatial_bounds:
-        comment: No repetition allowed.
+        comment: Recommended.
```
Comment (johtoblan) [inline py_mmd_tools/mmd_elements.yaml:207]: Earlier ( See Line 66) you use recommended: False.
This should probably be recommended: True
?

File: py_mmd_tools/mmd_elements.yaml [hunk 2]
```diff
@@ -385,12 +465,14 @@ platform:
...
         separator: ','
-        comment: Comma separated list.
+        comment: Recommended
+        description: Name of the platform(s) that supported the sensor used to create this dataset. Platforms can be of any type, including satellite, ship, station, aircraft or other. Indicate which controlled vocabulary that is used in the platform_vocabulary attribute. Comma separated list.
   resource:
     acdd:
       platform_vocabulary:
         separator: ','
-        comment: Comma separated list.
+        comment: Recommended
```
Comment (johtoblan) [inline py_mmd_tools/mmd_elements.yaml:474]: Same here

**Review by johtoblan — CHANGES_REQUESTED** [review]
Unsure if recommended: False and recommended: True are consistently used in mmd_elements.yml

File: py_mmd_tools/mmd_elements.yaml (hunk 1 above)
Comment (mortenwh) [reply py_mmd_tools/mmd_elements.yaml:207]: yes - I'll set this as an empty string

File: py_mmd_tools/mmd_elements.yaml (hunk 2 above)
Comment (mortenwh) [reply py_mmd_tools/mmd_elements.yaml]: Also changed to `comment:`

**Pull request merged**